atom/browser/default_app/index.html

```diff
@@ -58,7 +58,7 @@
 <body>
   <script>
     var execPath = require('remote').process.execPath;
-    var command = execPath + ' path-to-your-app';
+    var command = execPath + '--app=path-to-your-app';
```

Comment:
I think we should only make `--app` a hack for `webdriverio`, and the `electron path-to-your-app` should still be the preferred way of running apps.